opentelemetry-api/src/opentelemetry/context/base_context.py

```diff
@@ -99,6 +100,17 @@ def __getitem__(self, name: str) -> "object":
     def __setitem__(self, name: str, value: "object") -> None:
         self.__setattr__(name, value)
 
+    @contextmanager  # type: ignore
+    def __call__(
```

Comment:
We already have a global variable that is not a type but still uses `PascalCase`. I strongly suggest we choose a proper name for that method, like `set`, `with_values`, `with_`, instead of overloading `__call__`, which makes it look like we are constructing an instance of the Context class.

Reply:
Again we are having hard time naming things.
`with Context.with_values(...)` is very mouthful.
`Context.set` sounds like it is dealing with only one value rather than key-value pairs.

How about `Context.use(**kwargs)`?

Reply:
Fine with me. Or what about `set_scoped`? Or `scope`?

Reply:
+1 for Context.use()

Reply:
Let's go with `use`:

```suggestion
    def use(
```

Reply:
Updated to `Context.use`.